docs/tools/report-converter.md

```diff
@@ -11,6 +11,7 @@ a CodeChecker server.
   * [Address Sanitizer](#address-sanitizer)
   * [Memory Sanitizer](#memory-sanitizer)
   * [Thread Sanitizer](#thread-sanitizer)
+  * [Leak Sanitizer](#leak-sanitizer)
```

Comment:
Could you please update this doc too: https://github.com/Ericsson/codechecker/blob/master/docs/supported_code_analyzers.md#clang-sanitizers

![image](https://user-images.githubusercontent.com/6695818/124574262-94c7cb80-de4a-11eb-86ef-b7a1b6f9c053.png)

Reply:
@csordasmarton There you go! I have incorporated this change as well 😄